src/scrapers/mizrahi.ts

```diff
@@ -124,14 +129,22 @@ async function extractPendingTransactions(page: Page): Promise<Transaction[]> {
   });
 }
 
+async function postLogin(page: Page) {
+  await Promise.race([
+    waitUntilElementFound(page, afterLoginSelector),
+    waitUntilElementFound(page, invalidPasswordSelector),
+  ]);
+}
+
 class MizrahiScraper extends BaseScraperWithBrowser {
   getLoginOptions(credentials: ScraperCredentials) {
     return {
-      loginUrl: `${LOGIN_URL}`,
```

Comment:
Why was it like that?
(you don't need to answer, just note if I'm missing something.)